src/main/java/com/jagrosh/jmusicbot/commands/owner/PlaylistCmd.java

```diff
@@ -169,6 +176,75 @@ protected void execute(CommandEvent event)
             }
         }
     }
+
+    public class AppendlistqCmd extends OwnerCommand 
+    {
+        public AppendlistqCmd()
+        {
+            this.name = "appendq";
+            this.aliases = new String[]{"addq"};
+            this.help = "appends songs in queue to an existing playlist";
+            this.arguments = "<name>";
+            this.guildOnly = false;
+        }
+
+        @Override
+        protected void execute(CommandEvent event) 
+        {
+            String[] parts = event.getArgs().split("\\s+", 2);
+            if(parts.length<1)
+            {
+                event.reply(event.getClient().getError()+" Please include a playlist name to add!");
+                return;
+            }
+            String pname = parts[0];
```

Comment:
For consistency with the other playlist commands that also only take a list name as an argument (`make`, `delete` etc.), replace all spaces with an underscore.

![image](https://user-images.githubusercontent.com/39029839/133685189-2af9c008-2728-414a-a8f0-89dfadc0b911.png)

```suggestion
            String pname = event.getArgs().replaceAll("\\s+", "_")
```